packages/next/src/cli/next-test.ts

```diff
@@ -0,0 +1,195 @@
+import { writeFileSync } from 'fs'
+import { getProjectDir } from '../lib/get-project-dir'
+import { printAndExit } from '../server/lib/utils'
+import loadConfig from '../server/config'
+import { PHASE_PRODUCTION_BUILD } from '../shared/lib/constants'
+import {
+  hasNecessaryDependencies,
+  type MissingDependency,
+} from '../lib/has-necessary-dependencies'
+import { installDependencies } from '../lib/install-dependencies'
+import type { NextConfigComplete } from '../server/config-shared'
+import findUp from 'next/dist/compiled/find-up'
+import { findPagesDir } from '../lib/find-pages-dir'
+import { verifyTypeScriptSetup } from '../lib/verify-typescript-setup'
+import path from 'path'
+import spawn from 'next/dist/compiled/cross-spawn'
+
+export interface NextTestOptions {
+  testRunner?: string
+}
+
+export const SUPPORTED_TEST_RUNNERS_LIST = ['playwright'] as const
+export type supportedTestRunners = (typeof SUPPORTED_TEST_RUNNERS_LIST)[number]
```

Comment:
Sorry, to clarify, it should be `SupportedTestRunners`. I'm surprised we don't have a lint for this.